ccd-definition/CaseEventToFields/CareSupervision/manageOrders/5_orderDetailsSection.json

```diff
@@ -250,8 +250,47 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageOrders",
-    "CaseFieldID": "manageOrdersFurtherDirections",
+    "CaseFieldID": "manageOrdersIsByConsent",
     "PageFieldDisplayOrder": 20,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 5,
```

Comment:
PageDisplayOrder 6. I think it's not creating error since it's probably read at the beginning of the page